ui-v2/src/setupProxy.js

```diff
@@ -0,0 +1,10 @@
+const { createProxyMiddleware } = require("http-proxy-middleware");
+
+module.exports = function(app) {
```

Comment:
Where are we using this function and why do we have a hard-coded URL in it?

Reply:
Forgot to mention the doc: https://create-react-app.dev/docs/proxying-api-requests-in-development/#configuring-the-proxy-manually

> You do not need to import this file anywhere. It is automatically registered when you start the development server.

That's only for development. Hardcoded string not included in production build.

Reply:
Ah I see, it should be fine then